packages/web/src/dev/ValidatorsListRow.tsx

```diff
@@ -57,21 +61,26 @@ export interface CeloGroup {
 interface Props {
   group: CeloGroup
   expanded: boolean
+  onPinned: () => void
 }
 interface State {
   tooltip?: boolean
+  isPinned?: boolean
 }
 
 class ValidatorsListRow extends React.PureComponent<Props & I18nProps, State> {
   state = {
     tooltip: false,
+    isPinned: false,
   }
   tooltipRef = React.createRef<any>()
   removeDocumentListener: () => void
 
   constructor(...args) {
     super(...(args as [any]))
 
+    this.state.isPinned = this.isPinned()
```

Comment:
rather than setting in constructor its better to make a call to setState in the componentDidMount function